README.md

```diff
@@ -82,6 +82,17 @@ should be called to update the UI.
 This in turn will invoke the `build` method 
 which re-renders the widget.
 
+## Flutter application test
+
+As in all languages or platforms the secret to a successful application is to test it.
+Several tests should be done like the functionality test, the user experience test, which can help us correct any bug.</br>
+
+An application must have tested any function, any class or task needed to run correctly without errors.</br>
+
+In general, an application that wants to succeed must have performed several unit tests, these being the main ones since they test functions, classes alone.</br>
+Widget tests, as in Flutter there are several widgets also necessary to perform tests on each widget in order to understand if all of them are performing the intended function and if they are in the place where they should be.</br>
```

Comment:
41 word sentence. Are you going for some kind of personal record? 😜 
![image](https://user-images.githubusercontent.com/194400/74306045-531c1980-4d59-11ea-8f98-e34080563dc2.png)

But seriously, proof-read your work _before_ you assign it to me, the more of _other_ people's time you can save the more you will get paid. It's really that simple. Want to know the "secret formula" for getting rich? Save another wealthy person some time/effort. 💰

```suggestion
Widget tests are necessary to confirm that they are performing their intended function 
and correctly positioned in the layout.</br>
```
Same meaning. Less than _half_ the words.
![image](https://user-images.githubusercontent.com/194400/74306247-f10fe400-4d59-11ea-9796-6fce4d434f53.png)

Reply:
The sentences in the article were even bigger and I tried to reduce them to a minimum.